### PR TITLE
Update README.md

### LINKED ISSUE #5: Improve the README
## Do you have a favorite Git Resource?
Add it to the README!
 If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by andrew-scoppa**
Please  approve my changes!

Close #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# GitHub for Developers
+# GitHub for Developers and Project Leads
```
Comment (andrew-scoppa) [inline README.md:1]: Great thought

Comment (Fergdaddy16) [inline README.md:1]: Fergdaddy has approved because I am awesome

**Review by Fergdaddy16 — APPROVED** [review]

**Comment by Fergdaddy16** [comment]
approved

**Review by andrew-scoppa — COMMENTED** [review]
😺

File: README.md (hunk 1 above)
Comment (MonaCW) [reply README.md:1]: funny

**Pull request merged**